state_machine/modules/json_conf/include/klepsydra/state_machine/kpsr_state_machine.h

```diff
@@ -16,13 +16,13 @@
 
 #pragma once
 
+#include <klepsydra/state_machine/json_config_loader.h>
 #include <klepsydra/state_machine/sm_factory_impl.h>
 #include <klepsydra/state_machine/state_machine_listener.h>
-#include <klepsydra/state_machine/yaml_config_loader.h>
 
 namespace kpsr {
 namespace fsm {
-class FromYaml
+class FromJson
 {
 public:
     static std::shared_ptr<StateMachine> createStateMachine(const std::string &specPath);
```

Comment:
You can add a section on using these functions in the tutorial too. (but of course, it is not part of this PR)

Reply:
See commit [0025481](https://github.com/klepsydra-technologies/kpsr-tutorial/pull/37/commits/0025481223bbce6f210c4db08bff2da6357b0efb) at PR [KCE-65](https://github.com/klepsydra-technologies/kpsr-tutorial/pull/37)
```C++
auto stateMachine = kpsr::fsm::FromJson::createStateMachine(TEST_DATA "/sm1.json");
```
Is it enough? New example1b?

Reply:
Perfect.. I'll approve this PR, and the comments for tutorial will be in the respective PR.